### PR TITLE
Swagger: Fix `listTokensResponse`

### PR DESCRIPTION
**Pull request by julienduchesne**
It should return a list of Tokens, not a single one 

Also regenerated the API spec from the latest changes + this branch

### CODE REVIEW

File: pkg/services/serviceaccounts/api/token.go [hunk 1]
```diff
@@ -246,7 +246,7 @@ type DeleteTokenParams struct {
 // swagger:response listTokensResponse
 type ListTokensResponse struct {
 	// in:body
-	Body *TokenDTO
+	Body []*TokenDTO
```
Comment (gamab) [inline pkg/services/serviceaccounts/api/token.go:249]: Good job spotting this. I believe:
```suggestion
	Body []TokenDTO
```

**Review by gamab — APPROVED** [review]
LGTM. Left a small comment :)

**Review by papagian — APPROVED** [review]

File: public/api-enterprise-spec.json [hunk 2]
```diff
@@ -685,216 +685,6 @@
         }
       }
     },
-    "/datasources/{datasourceId}/disable-permissions": {
```
Comment (papagian) [inline public/api-enterprise-spec.json:688]: these changes come from another repository: [DatasourceACL: Remove deprecated datasource permissions endpoints](https://github.com/grafana/grafana-enterprise/pull/5880)
we need to introduce a similar [swagger-gen](https://drone.grafana.net/grafana/grafana/146375/7/3) step there too

Comment (gamab) [reply public/api-enterprise-spec.json:688]: Yes, I was a bit confused by this. But I looked at the other repo and these endpoints were removed.

File: pkg/services/serviceaccounts/api/token.go (hunk 1 above)
Comment (julienduchesne) [reply pkg/services/serviceaccounts/api/token.go:249]: Hmm, I looked at some other responses and they had pointers as well.

Also, I generated the OpenAPI client from my PR and it worked well

So, I'm not sure. I personally would lean towards the pointer version

Comment (gamab) [reply pkg/services/serviceaccounts/api/token.go:249]: Alright I wonder the reasoning behind it. It does look a bit strange to me but in the end the json does not return pointers anyway :shrug: 

Should the result be aligned then?
https://github.com/grafana/grafana/blob/ff96532dd06085e864983f20bd7cae4f0c1e5ad4/pkg/services/serviceaccounts/api/token.go#L81

Comment (julienduchesne) [reply pkg/services/serviceaccounts/api/token.go]: I removed the pointer. Makes no difference in Swagger, so let's use that!

**Pull request merged**